### PR TITLE
MgmtSystem Quality migration to Odoo 9.0

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

**Comment by dreispt** [comment]
:+1:

File: mgmtsystem_quality/README.rst [hunk 1]
```diff
@@ -74,6 +74,7 @@ Contributors
 ------------
 
 * Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
```
Comment (max3903) [inline mgmtsystem_quality/README.rst:79]: @naousse Please add:

```
* Luk Vermeylen <luk@allmas-it.be>
* Daniel Reis <dgreis@sapo.pt>
* Maxime Chambreuil <mchambreuil@ursainfosystems.com>
```

File: mgmtsystem_quality/README.rst [hunk 2]
```diff
@@ -73,7 +73,9 @@ Credits
 Contributors
 ------------
 
-* Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
```
Comment (max3903) [inline mgmtsystem_quality/README.rst:76]: @naousse Please keep this one.

**Comment by max3903** [comment]
:+1:

**Pull request merged**